app/models/responses.go

```diff
@@ -95,6 +99,9 @@ func (r *ResultMessage) MarshalXML(e *xml.Encoder, start xml.StartElement) error
 	e.EncodeElement(r.MessageId, xml.StartElement{Name: xml.Name{Local: "MessageId"}})
 	e.EncodeElement(r.ReceiptHandle, xml.StartElement{Name: xml.Name{Local: "ReceiptHandle"}})
 	e.EncodeElement(r.MD5OfBody, xml.StartElement{Name: xml.Name{Local: "MD5OfBody"}})
+	if r.MessageAttributes != nil {
+		e.EncodeElement(r.MD5OfMessageAttributes, xml.StartElement{Name: xml.Name{Local: "MD5OfMessageAttributes"}})
+	}
```

Comment:
Here is the fix for Bug 3 "MD5OfMessageAttributes is missing on the message with message attributes from SQS-ReceiveMessages".

Reply:
I'm not sure why I left this out.  I think there was a reason though.  A lot of this had to do with the raw vs regular message handling.  Can you tell me more about what your issues are on this?